pkgs/top-level/all-packages.nix

```diff
@@ -4184,6 +4184,13 @@ in
     enableNpm = false;
     openssl = openssl_1_1;
   };
+  nodejs-latest = callPackage ../development/web/nodejs/v12.nix {
+    openssl = openssl_1_1;
+  };
```

Comment:
Would probably be neater to do this as in https://github.com/NixOS/nixpkgs/blob/86bd1d6dedb271f59e9ef42fa8da2ad2521aab0d/pkgs/top-level/all-packages.nix#L15428